library/pom.xml

```diff
@@ -325,6 +325,11 @@
       <version>7.0.6</version>
     </dependency>
     <dependency>
+      <groupId>com.yahoo.datasketches</groupId>
```

Comment:
@sandeep-n Is there any other functionality that you envision to build with this dependency?